client/components/like-button/button.jsx

```diff
@@ -83,8 +83,14 @@ var LikeButton = React.createClass( {
 
 		containerClasses = classnames( containerClasses );
 
+		if ( likeCount === 0 && ! this.props.showCount ) {
+			likeCount = '';
+		} else {
+			likeCount = likeCount + ' ';
```

Comment:
sure, but what's the proper `px` for 1 space? I'd rather just use a space because a space is what's called for.